.github/workflows/ci.yml

```
@@ -42,3 +42,28 @@ jobs:
       run: |
         chmod -R ugo+w .
         make -f misc/docker-ci/check.mk ${{ matrix.task }}
+
+  fuzz:
```

"fuzz" conflicts with our fuzz build, so I'd name otherwise, like:

```suggestion
  # See https://github.com/google/oss-fuzz/tree/master/projects/h2o for details
  ossfuzz:
    name: OSS Fuzz
```